Dependencies/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ set(BGFX_USE_DEBUG_SUFFIX OFF CACHE BOOL "Add 'd' suffix to debug output targets
 add_subdirectory(bgfx.cmake)
 target_compile_definitions(bgfx PRIVATE BGFX_CONFIG_DEBUG_UNIFORM=0)
 target_compile_definitions(bgfx PRIVATE BGFX_CONFIG_MULTITHREADED=1)
-target_compile_definitions(bgfx PRIVATE BGFX_CONFIG_MAX_VERTEX_STREAMS=32)
+target_compile_definitions(bgfx PRIVATE BGFX_CONFIG_MAX_VERTEX_STREAMS=16)
```

Comment:
why?

Reply:
The crash happens here with 32 streams but works ok with 16. https://github.com/bkaradzic/bgfx/blob/16ee93b88d7ec056a4cc5a7a7d9679af1e8cfefe/src/renderer_d3d12.cpp#L4292 I tried to find some documentation/limits but I didn't find anything.

Reply:
I (eventually) had idea to use dxcpl with validation layer and got this:
```
D3D12 ERROR: ID3D12CommandList::IASetVertexBuffers: StartSlot(0) + NumDescriptors(33) must not be greater than 32. [ EXECUTION ERROR #726: SET_VERTEX_BUFFERS_INVALID]
```
BGFX_CONFIG_MAX_VERTEX_STREAMS=31 should be ok.

Reply:
We talked offline and verified 16 is okay. Plus, we limit to 16 on the JS side right now anyways.